dotnet/src/Experimental/Process.Abstractions/KernelProcess.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+
+namespace Microsoft.SemanticKernel;
+
+/// <summary>
+/// A serializable representation of a Process.
+/// </summary>
+public sealed class KernelProcess : KernelProcessStep<KernelProcessState>
+{
+    /// <summary>
+    /// The collection of Steps in the Process.
+    /// </summary>
+    public IList<KernelProcessStepBase> Steps { get; init; }
```

Comment:
Minor - The constructor requires this property is initialized.  What is the case for using `init` also?  (If declared only with `get`, it will be readonly...which only affects the collection reference.  The collection itself can be modified unless its `IReadOnlyList<T>`.)

Reply:
Agree, updated.